packaging/baker2022/inverted_distortion_data_assembly.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+from brainio.assemblies import BehavioralAssembly
+from brainio.packaging import package_data_assembly
+import pandas as pd
+
+'''
+Experiment Information:
+https://www.sciencedirect.com/science/article/pii/S2589004222011853#sec9
+ - (Data/subjects are from experiment 1 in paper above)
+
+ - 32 subjects 
+ - 40 images for each condition (whole, fragmented, frankenstein) = 120 images/subject shown
+ - 120 * 32 = 3840 total images shown 
+ - However there are only 3706 trials total, for data issues (according to Baker)
+ - 9-way AFC, from set: {bear, bunny, cat, elephant, frog, lizard, tiger, turtle, wolf}
+ 
+'''
+
+# initial csv to dataframe processing:
+all_subjects = pd.read_csv('human_data/inverted.csv')
+
+
+# construct the assembly
+assembly = BehavioralAssembly(all_subjects['RSP'],
```

Comment:
do we have any information about the actual behavioral response of the subject? If it's correct, we can infer it from `truth`, but if it's incorrect, do we know what subjects responded?